website/common/script/i18n.js

```diff
@@ -60,4 +61,26 @@ function t (stringName) {
   }
 }
 
+// translates passed strings to two (passed) languages and concatenates them
+// returns an array
+function translateTwice (languages, messages) {
```

Comment:
Currently it has 3 users (one more if a test is converted). I took a glance if any similar reports are open, but there are none. Since this handles both gifted subs and gems, I think all automated two user interactions are handled, so you'd be correct.